tests/compatibility-test.py

```diff
@@ -7,17 +7,20 @@
 import string
 import docker
 
-from kuberay_utils import utils
+import kuberay_utils.utils as utils
```

Comment:
Just curious -- could you explain this change?
(I don't really understand Python's import system.)

Reply:
In my understanding (not 100% sure), they are equivalent in Python. 

I am not sure about Python's conventions.

`from kuberay_utils import utils`: 
* `kuberay_utils` is a package (Package: A collection of modules.)
* `utils` is a module (Module: A single python script.)

Then, we can use `utils.func()` to call a function.

However, users can also use `from ${module} import ${function}`. Then, we can call the function via `func()` directly.

Hence, I hope describe that `utils` is a module rather than a function by `import kuberay_utils.utils as utils`, a more explicit way.

This change is not necessary. Do we need to keep the original one?

Reply:
A good article: https://towardsdatascience.com/understanding-python-imports-init-py-and-pythonpath-once-and-for-all-4c5249ab6355